retesteth/configs/clientconfigs/evmone.cpp

```diff
@@ -274,42 +275,66 @@ string const evmone_config = R"({
+      "EOF_UnreachableCodeSections" : "err: unreachable_code_sections",
+      "EOF_UndefinedInstruction": "err: undefined_instruction",
+      "EOF_TruncatedImmediate": "err: truncated_instruction",
+      "EOF_InvalidJumpDestination": "err: invalid_rjump_destination",
+      "EOF_TooManyCodeSections": "err: too_many_code_sections",
       "EOF_InvalidTypeSectionSize": "err: invalid_type_section_size",
       "EOF_InvalidFirstSectionType": "err: invalid_first_section_type",
-      "EOF_TooManyCodeSections": "err: too_many_code_sections",
+      "EOF_InvalidMaxStackHeight": "err: invalid_max_stack_height",
+      "EOF_InvalidCodeTermination": "err: no_terminating_instruction",
+      "EOF_ConflictingStackHeight": "err: stack_height_mismatch",
+      "EOF_InvalidNumberOfOutputs": "err: stack_higher_than_outputs_required",
+      "EOF_MaxStackHeightExceeded": "err: max_stack_height_above_limit",
+      "EOF_UnreachableCode": "err: unreachable_instructions",
+      "EOF_InputsOutputsNumAboveLimit": "err: inputs_outputs_num_above_limit",
+      "EOF_StackUnderflow": "err: stack_underflow",
+      "EOF_StackOverflow": "err: stack_overflow",
       "EOF_InvalidCodeSectionIndex": "err: invalid_code_section_index",
-      "EOF_UndefinedInstruction": "err: undefined_instruction",
-      "EOF_ZeroSectionSize": "err: zero_section_size",
-      "EOF_NonEmptyStackOnTerminatingInstruction": "err: non_empty_stack_on_terminating_instruction",
-      "EOF_InvalidSectionBodiesSize": "err: invalid_section_bodies_size",
+      "EOF_InvalidDataloadnIndex": "err: invalid_dataloadn_index",
+      "EOF_JumpfDestinationIncompatibleOutputs": "err: jumpf_destination_incompatible_outputs",
+      "EOF_CallfToNonReturningFunction": "err: callf_to_non_returning_function",
+      "EOF_TooManyContainerSections": "err: too_many_container_sections",
+      "EOF_InvalidContainerSectionIndex": "err: invalid_container_section_index",
+      "EOF_EofCreateWithTruncatedContainer": "err: eof_create_with_truncated_container",
+      "EOF_InvalidNonReturningFlag" : "err: invalid_non_returning_flag",
       "PostParisUncleHashIsNotEmpty" : "block.uncleHash != empty",
       "PostParisDifficultyIsNot0" : "block.difficulty must be 0"
     }
 })";
 
 string const evmone_start = R"(#!/bin/sh
 
-wevm=$(which evmone)
+wevm=$(which evmone-t8n)
 if [ -z $wevm ]; then
-   >&2 echo "Can't find EvmOne's 'evmone' executable alias in the system path!"
+   >&2 echo "Can't find EvmOne's 'evmone-t8n' executable alias in the system path!"
    exit 1
 fi
 
-if [ $1 = "eof" ] || [ $1 = "t8n" ] || [ $1 = "b11r" ]; then
-    evmone $1 $2 $3 $4 $5 $6 $7 $8 $9 $10 $11 $12 $13 $14 $15 $16 $17 $18 $19 $20 $21 $22 $23 $24 $25 $26
+if [ $1 = "t8n" ] || [ $1 = "b11r" ]; then
+    evmone-t8n $2 $3 $4 $5 $6 $7 $8 $9 $10 $11 $12 $13 $14 $15 $16 $17 $18 $19 $20 $21 $22 $23 $24 $25 $26
 elif [ $1 = "-v" ]; then
-    evmone -v
+    evmone-t8n -v
+elif [ $1 = "eof" ]; then
+    result=$(echo $5 | evmone-eofparse)
```

Comment:
Passing number 5 here is bot reliable. Its position might change. What are the 2, 3, 4 that ignored?